### PR TITLE
docs(test): improved documentation

### PR DESCRIPTION
**Pull request by anshulwadhawan**
In response to issue #686.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -245,6 +245,12 @@ docker run jinaai/jina:my-local-version check
 
 It prints a list of components the current version of Jina supports, and then exits. Make sure yours are not in red.
 
+You need to build a local docker image tagged jinaai/jina:test-pip for all the tests to run as in the CI, via:
```
Comment (JoanFM) [inline CONTRIBUTING.md:248]: I would move this before the instructions to run `pytests`

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (anshulwadhawan) [reply CONTRIBUTING.md:248]: Sure

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -225,6 +225,12 @@ We don't enforce naming of PRs and branches, but we recommend you follow the sam
 <a name="-testing-jina-locally-and-on-ci"></a>
 ## 💥 Testing Jina Locally and on CI
 
+You need to build a local docker image tagged jinaai/jina:test-pip for all the tests to run as in the CI, via:
```
Comment (cristianmtr) [inline CONTRIBUTING.md]:
```suggestion
You need to build a local docker image tagged `jinaai/jina:test-pip` for all the tests to run as in the CI, via: 
```

**Comment by cristianmtr** [comment]
I assume this is about [this issue](https://github.com/jina-ai/jina/issues/686) 

I see that you asked about working on this.

Maybe refer to it in the description of this PR.

**Comment by anshulwadhawan** [comment]
> I assume this is about [this issue](https://github.com/jina-ai/jina/issues/686)
> 
> I see that you asked about working on this.
> 
> Maybe refer to it in the description of this PR.

File: CONTRIBUTING.md (hunk 2 above)
Comment (anshulwadhawan) [reply CONTRIBUTING.md]: Sure. Made the changes.

**Review by JoanFM — APPROVED** [review]

**Comment by JoanFM** [comment]
Thank you for your contribution @anshulwadhawan! Really appreciated!!

**Pull request merged**